lib/aptible/api/aws_instance.rb

```diff
@@ -4,17 +4,29 @@ class AwsInstance < Resource
       belongs_to :stack
       belongs_to :ami
       has_many :operations
+      has_many :instance_layer_members
       embeds_many :databases
 
       field :id
       field :instance_id
       field :instance_type
       field :availability_zone
       field :name
-      field :layers
       field :status
       field :created_at, type: Time
       field :updated_at, type: Time
+
+      def stack_layers
+        instance_layer_members.map(&:links)
```

Comment:
I tried running a provision operation: https://my.papertrailapp.com/groups/617523/events?q=source%3Atest-ruby+operation-2&focus=1330036422400487501&selected=1330036422400487501

It failed out on this line, I'm guessing because that model is now `instance_layer_memberships`.